tests/java/test_appengine_standard.py

```diff
@@ -17,6 +17,7 @@
 
 from ..common.common import Common
 
+
```

Comment:
nit: not needed space IMHO

Reply:
Yes, they were added by the same tool (black) that was causing the lint test to fail.  I just ran the formatter over the files and this is what it did. 🤷‍♂️